actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,31 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
+        delegate :==, :each, :empty?, to: :@variants
```

Comment:
Couldn't find any public apps on Github using #==, so I just removed it. I don't think it's worth a deprecation cycle.